### PR TITLE
openSUSE Leap Micro as container host

### PR DESCRIPTION
**Pull request by keichwa**
**For rendered output, see beginning of description of:** https://github.com/SUSE/spacewalk/issues/25126

https://github.com/SUSE/spacewalk/issues/25126

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/installation-and-upgrade/nav-installation-and-upgrade-guide.adoc [hunk 1]
```diff
@@ -30,7 +30,7 @@ ifeval::[{uyuni-content} == true]
 **** xref:install-uyuni.adoc[Install Uyuni Server with openSUSE]
 *** Proxy
 **** xref:container-deployment/uyuni/proxy-container-setup-uyuni.adoc[Proxy Preparation]
-**** xref:container-deployment/uyuni/proxy-deployment-uyuni.adoc[Deploy {productname} Proxy]
+**** xref:container-deployment/uyuni/proxy-deployment-uyuni.adoc[Deploy on {leap}]
```
Comment (aaannz) [inline modules/installation-and-upgrade/nav-installation-and-upgrade-guide.adoc:33]:
```suggestion
**** xref:container-deployment/uyuni/proxy-deployment-uyuni.adoc[Deploy on {leapmicro}]
```

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc [hunk 2]
```diff
@@ -69,124 +69,99 @@ This table shows the hardware requirements for deploying {productname} Proxy.
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc:69]: Please remove those two paths above and only have one path:
```
[path]``/var/lib/containers/storage/volumes``
| Minimum 100 GB, Storage requirements should be calculated for the number of different clients and OS images usage.
```

File: modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc [hunk 3]
```diff
@@ -69,124 +69,99 @@ This table shows the hardware requirements for deploying {productname} Proxy.
...
-====
-Alternatively you may install [systemitem]``mgrpxy-zsh-completion`` or [systemitem]``mgrpxy-fish-completion``.
-====
+. Install the container tools:
 +
 
 [source, shell]
 ----
-transactional-update pkg install mgrpxy mgrpxy-bash-completion uyuni-storage-setup-proxy
+zypper in mgradm mgrctl mgradm-bash-completion mgrctl-bash-completion netavark uyuni-storage-setup-server
```
Comment (aaannz) [inline modules/installation-and-upgrade/pages/container-deployment/uyuni/proxy-deployment-uyuni.adoc:129]:
```suggestion
zypper in mgrpxy mgrpxy-bash-completion uyuni-storage-setup-proxy
```

**Review by aaannz — CHANGES_REQUESTED** [review]

**Comment by keichwa** [comment]
Thanks for reviewing!  Feedback plus minor edits are in HTML is updated (see chained link in description).  More reviews appreciated.  Then I'll merge it tomorrow in the morning.

**Review by aaannz — APPROVED** [review]
👍🏾

**Pull request merged**